### ISSUE TITLE
Conversion to Es6

### ISSUE THREAD

**Issue by 1fabiopereira** [voltraco/dml#6]
I wonder if there is any problem before we start implementing the functionality described in #5  to convert your entire base to es6 and a more modular formatting for the functions?

**Comment by heapwolf** [comment]
Currently I'm trying to limit my use of es6 to what is supported by node.js and the most popular browsers. The main reason for this is to keep the tool-chain requirements minimalistic (this is the philosophy i've maintained for all the @voltraco work, so far) That said, there might things that can be brought up to date?

**Comment by 1fabiopereira** [comment]
Yes, we can use features that do not break the compatibility, like arrow function, class, conts, let etc ... are simple changes that will make the syntax leaner and improve the performance of the plugin, what do you think? If you'd rather not for me without problems.

**Comment by heapwolf** [comment]
Huge +1 👍

**Comment by 1fabiopereira** [comment]
Beauty in a few weeks I'll send a pull request with this change after we start to figure out how we'll do the formatters plugin, it's going to take about 3 weeks. I'm in the middle of a project and I have to conclude it first, but in the meantime I'm already thinking about the best approach. :smile:
